### PR TITLE
ARGO-2422 Move common parameters to separate config file

### PR DESCRIPTION
**Pull request by eimamagi**
Idea is to make ncg.conf as generic as possible and move common variables to separate file.

### CODE REVIEW

File: config/ncg-vars.conf [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# WEBAPI_TOKEN = MY_WEBAPI_TOKEN
+
+# List of metric profiles used for configuring
+# ARGO Monitoring Engine
+# (default: )
+# PROFILES = MON 
+
+# ARGO POEM URL
+# (default: https://poem.egi.eu)
+# POEM_ROOT_URL = https://poem.egi.eu
```
Comment (kkoumantaros) [inline config/ncg-vars.conf]: default should be poem.argo.grnet.gr

Comment (eimamagi) [reply config/ncg-vars.conf]: Problem is that poem.argo.grnet.gr is superpoem and I need a tenant POEM URL.

Comment (themiszamani) [reply config/ncg-vars.conf]: Add the raining Tenant .

**Comment by kkoumantaros** [comment]
But not the storm one

**Pull request merged**